platform.py

```
@@ -139,6 +140,47 @@ def is_internet_available():
+                    delay = 2 ** (attempt + 1)
+                    logger.warning(
+                        "Package download failed: %s. Retrying in %ds... (attempt %d/%d)",
+                        e, delay, attempt + 1, max_retries,
+                    )
+                    try:
+                        if hasattr(self, "_http_response") and self._http_response is not None:
+                            self._http_response.close()
+                        if hasattr(self, "_http_session"):
+                            self._http_session.close()
+                    except Exception:
+                        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)